queue_job/models/queue_job.py

```diff
@@ -305,6 +305,10 @@ def related_action_open_record(self):
             )
         return action
 
+    @job
+    def _test_job(self):
+        _logger.debug("Running test job.")
```

Comment:
Maybe worth to use `info`? Since we create this job on purpose, it's worth to have more visibility here (even if we'll see if the job is done or not eventually).

Reply:
makes sense, fixed